website/docs/secrets/getting-started.mdx

```diff
@@ -2,3 +2,188 @@
 title: Getting Started
 hide_title: true
 ---
+
+import TierLabel from "./../_components/TierLabel";
+import AlphaWarning from "../_components/_alpha_warning.mdx";
+
+# Getting started with secrets management <TierLabel tiers="Enterprise" />
+
+<AlphaWarning/>
+
+This guide shows you a basic experience to get started with Weave Gitops Secrets.
+It covers the scenario of setting up the capability in a test environment and how to use it for your applications.
+
+## Requirements
+- You have a test Weave Gitops Enterprise environment with Flux installed.
+- You have a secret in AWS secrets manager.
+
+## Add the secrets infra
+
+In order to allow you managing external secrets stores and secrets, install `external-secrets` profile
+from `weaveworks-charts` profiles repository.
+
+<details><summary>Expand to see example</summary>
```

Comment:
we usually point the user to to adding the profile through adding an application in the UI for existing clusters, or selecting a profile during cluster creation. I find sharing the helm files like this directly a bit confusing. 
We sometimes elaborate by explaining a certain part of the configuration in the values file.
 this is how it was done for the policy https://docs.gitops.weave.works/docs/policy/getting-started/#install-policy-profile and Waleed did something similar in his secrets docs as well